src/plugins/custom_integrations/public/plugin.tsx

```diff
@@ -48,16 +46,7 @@ export class CustomIntegrationsPlugin
   ): CustomIntegrationsStart {
     const services = servicesFactory({ coreStart, startPlugins });
 
-    const languageClientsUiComponents = new Map<string, React.FC>();
-
-    // Set the language clients components to render in Fleet plugin under Integrations app
-    // Export component only if the integration has exportLanguageUiComponent = true
-    languageIntegrations
-      .filter((int) => int.exportLanguageUiComponent)
-      .map((int) => {
-        const ReadmeComponent = () => <OverviewComponent packageName={`${int.id}`} />;
-        languageClientsUiComponents.set(`language_client.${int.id}`, ReadmeComponent);
-      });
```

Comment:
I replaced this mapping logic with a simple static object. Seemed a bit easier to reason about when "opting in" to the custom README logic.